ui/translations/BackendSelect_fr.ts

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr_FR">
+    <context>
+        <name>BackendSelect</name>
+        <message>
+            <location line="66" filename="../BackendSelect.qml"/>
+            <source>Select Your Backend</source>
+            <translation>Sélectionnez votre backend</translation>
+        </message>
+        <message>
+            <location line="101" filename="../BackendSelect.qml"/>
+            <source>A backend provides services used by OpenVoiceOS Core</source>
+            <translation>Un backend fournit des services utilisés par OpenVoiceOS Core</translation>
+        </message>
+        <message>
+            <location line="119" filename="../BackendSelect.qml"/>
+            <source>Backend</source>
+            <translation>Backend</translation>
+        </message>
+        <message>
+            <location line="130" filename="../BackendSelect.qml"/>
+            <source>Personal Backend</source>
+            <translation>Back-end personnel</translation>
```

Comment:
My only comment, for consistency, you should use `backend` or `back-end` not the both of them.

Reply:
this was also fixed